cloudevents/primer.md

```diff
@@ -143,7 +143,7 @@ values that would be used, just like any other event it might generate.
 Since not all event producers generate their events as CloudEvents, there are
 a set of [adapters](./adapters.md) defined that show how to map events from
 some popular event producers into CloudEvents. These adapters are non-normative
-but are the specification authors' best guess as to how the CloudEvents 
+but are the specification authors' best guess as to how the CloudEvents
```

Comment:
As I was preparing v1.0.2-rc I noticed the trailing space so I slipped in a fix here.